.github/workflows/test.yml

```diff
@@ -45,8 +48,11 @@ jobs:
       matrix:
         python: ['3.8', '3.9']
         django: ['3.1', '3.2']
-        wagtail: ['2.14']
+        wagtail: ['2.14', '2.15', '2.16']
```

Comment:
**Question :** is it worth testing against all wagtail versions here ? Or is it safe to assume it is already tested against them in the previous job ?